lib/src/key/mod.rs

```diff
@@ -4,26 +4,28 @@
 /// crate::key::root::hb                 /!hb{ts}/{nd}
 /// crate::key::root::nd                 /!nd{nd}
 /// crate::key::root::ni                 /!ni
-/// crate::key::root::ns                 /!ns{ns}
+/// crate::key::root::ns                 /!ns{ns name}
 ///
 /// crate::key::node::all                /${nd}
 /// crate::key::node::lq                 /${nd}!lq{lq}{ns}{db}
 ///
 /// crate::key::namespace::all           /*{ns}
-/// crate::key::namespace::db            /*{ns}!db{db}
-/// crate::key::namespace::di            /+{ns id}!di
+/// crate::key::namespace::db            /*{ns}!db{db name}
```

Comment:
This is basically https://github.com/surrealdb/surrealdb/pull/2554/files#r1311674064 for dbs.